src/tasks/actions/thunks.ts

```diff
@@ -523,10 +523,13 @@ export const saveNewScript = (script: string, preamble: string) => async (
     dispatch(notify(copy.taskCreatedSuccess()))
     dispatch(checkTaskLimits())
   } catch (error) {
+    console.error('error while saving task script: ', error)
+    const message = getErrorMessage(error)
+    console.error(`error message: ${message}`)
     if (isLimitError(error)) {
+      console.error('is limit error')
```

Comment:
We don't want `console.error` calls - they lower our lighthouse scores: https://web.dev/errors-in-console/ and cause warnings to show up in jest tests

Reply:
i added them because i couldn't tell why the test was failing: the notification error is unreadable in the video (the text is too grainy).  

you still want me to remove it?  (I at least want to wait until the ci run is over ; to keep that in until the tests pass)

Reply:
(it did pass, btw)